### PR TITLE
Hotfix to get github actions working (no more error for failed unit test)

### PR DESCRIPTION
**Pull request by SamuelDegelia-NOAA**
Quick fix for #195 via @CoryMartin-NOAA. This should stop the error for a failed unit test when submitting a PR or adding a new commit.

### CODE REVIEW

**Review by guoqing-noaa — APPROVED** [review]

**Comment by guoqing-noaa** [comment]
@SamuelDegelia-NOAA Thanks for the quick fix.

This is a hotfix, so I will merge it right now.

File: .github/workflows/unittests.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ on: [push, pull_request]
 
 jobs:
   ctests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```
Comment (TingLei-NOAA) [inline .github/workflows/unittests.yaml:6]: @SamuelDegelia-NOAA  Could you explain why the changes of the virtual machine type would resolve this issue?

Comment (SamuelDegelia-NOAA) [reply .github/workflows/unittests.yaml:6]: @TingLei-NOAA I am not sure why it fixes it. The error was related to pip, so I am guessing that there was an update to the latest default ubuntu version that broke pip. @CoryMartin-NOAA might have more information.

Comment (CoryMartin-NOAA) [reply .github/workflows/unittests.yaml:6]: @SamuelDegelia-NOAA 's hunch is the same as mine. I'd rather just deal with this later when that version of the virtual machine is no longer available.

Comment (TingLei-NOAA) [reply .github/workflows/unittests.yaml:6]: @SamuelDegelia-NOAA @CoryMartin-NOAA  Thanks for this explanation.
